zuul.d/pipelines.yaml

```diff
@@ -176,12 +176,12 @@
 
 - pipeline:
     name: periodic
-    description: Jobs in this queue are triggered on a timer. UTC-0 0:00, 8:00 and 16:00
+    description: Jobs in this queue are triggered on a timer. UTC-0 0:00, 04:00, 08:00 and 16:00
```

Comment:
do we really need to run periodic jobs 4 times per day ? looks like a bit more frequency. actually, I personally think the previous 3 times per day even more frequency :)

Reply:
Three times in one day 04:00, 08:00 and 16:00 , it's ok?

Reply:
yes, I prefer one day 04:00, 08:00 and 16:00, as it is only for us to conveniently to check the results in working time. otherwise, I think 1 time per day is enough. it is impossible the tested projects code changed many times every day.